thriftserver/thriftserver/base/thriftserver-db-pvc.yaml

```diff
@@ -0,0 +1,11 @@
+---
+kind: PersistentVolumeClaim
+apiVersion: v1
+metadata:
+  name: thriftserver-db
+spec:
+  accessModes:
+    - ReadWriteOnce
+  resources:
+    requests:
+      storage: 1Gi
```

Comment:
Do we have any understanding if this is a reasonable value? (I am not sure what is actually stored in the DB)

Reply:
This storage size request comes from Internal DH, where the 1Gi is sufficient. Ti stores metadata about the SQL jobs spark is tasked to perform, who are the connected clients, etc...

I have no idea how it can grow in bigger deployments that that. @maulikjs, do you think this value is reasonable?

Reply:
I don't think we've seen issues due to the DB filling up before.

Reply:
Ok